app/controllers/admin/editions_controller.rb

```diff
@@ -207,9 +207,13 @@ def get_layout
     end
   end
 
-  def fetch_version_and_remark_trails
-    @document_remarks = Document::PaginatedRemarks.new(@edition.document, params[:remarks_page])
-    @document_history = Document::PaginatedHistory.new(@edition.document, params[:page])
+  def fetch_version_and_remark_trails(preview_design_system: false)
+    if preview_design_system
```

Comment:
Does `preview_design_system` need to be defined as a parameter on this method?

Couldn't the method call `preview_design_system?(next_release: false)` directly itself?

```suggestion
  def fetch_version_and_remark_trails
    if preview_design_system?(next_release: false)
```